modules/security-understanding-container-validation.adoc

```diff
@@ -0,0 +1,157 @@
+
+. The cluster downloads an image manifest and calculates its digest.
+
+. The cluster compares the calculated digest of the image manifest with the known digest.
+If the digest values do not match, the image is rejected without further processing.
+
+. The cluster downloads an image config file, which contains metadata such as SHA256 digests for each part of the container image, and calculates the digest of the image config file.
+
+. The cluster compares the calculated digest of the image config file with the digest that is provided in the image manifest.
+If the digest values do not match, the image is rejected without further processing.
+
+. For each file that comprises the container image, the cluster downloads the file and calculates its digest.
+
+. The cluster compares the calculated digest of each file that makes up the image with the corresponding digest that is provided in the image manifest.
+If the digest values do not match, the image is rejected.
+
+Once the cluster completes all of the previous steps, the integrity of the image has been validated.
+
+[NOTE]
+====
+The image manifest is validated only when the manifest digest is known, which is true in the following cases:
+
+. The image is pulled by specifying the SHA256 image digest, for example `quay.io/openshift-release-dev/ocp-release@sha256:ea7ac3ad42169b39fce07e5e53403a028644810bee9a212e7456074894df40f3`.
+
+. The image signature is also being validated. See “Image authenticity validation” for more information.
+====
+
+[id="security-understanding-image-authenticity_{context}"]
+== Image authenticity validation
+
+In addition to validating the integrity of images, {product-title} clusters also validate their authenticity by certifying that the images were approved by a trusted source, in this case Red{nbsp}Hat.
+This validation process includes checking the digital signature of the images.
+
+Cluster administrators can use `ClusterImagePolicy` resources to declare image policy for the cluster.
+
+:FeatureName: Configuring cluster image policies
+include::snippets/technology-preview.adoc[]
+
+.Example `ClusterImagePolicy` resource 1
+[source,yaml]
+----
+kind: ClusterImagePolicy
+metadata:
+  name: mypolicy-0
+spec:
+  scopes:
+  - test0.com
+  policy:
+    rootOfTrust:
+      policyType: FulcioCAWithRekor
+      fulcioCAWithRekor:
+        fulcioCAData: dGVzdC1jYS1kYXRhLWRhdGE=
+        fulcioSubject: <example_input_here> <1>
+        rekorKeyData: dGVzdC1yZWtvci1rZXktZGF0YQ==
+    fulcioSubject:
+      oidcIssuer: https://OIDC.example.com
+      signedEmail: test-user@example.com
+    signedIdentity:
+      matchPolicy: RemapIdentity
+      remapIdentity:
+        prefix: test-remap-prefix
+        signedPrefix: test-remap-signed-prefix
+
+----
+<1> Specifies OIDC issuer and the email of the Fulcio authentication configuration.
+
+.Example `ClusterImagePolicy` resource 2
+[source,yaml]
+----
+kind: ClusterImagePolicy
```

Comment:
```suggestion
apiVersion: config.openshift.io/v1alpha1
kind: ClusterImagePolicy
```

Reply:
Sounds good, does this need to be added to the other example too?

Reply:
yeah, all the ClusterImagePolicy YAML examples will need this `apiVersion` line